Jenkinsfile

```diff
@@ -69,6 +69,39 @@ pipeline {
         stage('OIDC Authenticator') {
           steps { sh 'ci/test cucumber_authenticators_oidc' }
         }
+        stage('Allocate Azure Authenticator Instance'){
+          steps {
+            script {
+              node('azure-authn'){
```

Comment:
not 💯 on what this is doing- is this setting up the environment?

Reply:
this just runs separate code in the context of a workspace?

Reply:
I added a comment on this in the commit. Still confused?

Reply:
In the comment `one` should have an uppercase O (:

Reply:
What do you mean when you say "Allocating an Azure VM"? As in getting the VM up and configuring it to be ready for running tests against?

Reply:
without the word "again". it is a new machine.